src/Select.js

```diff
@@ -208,9 +208,12 @@ const Select = React.createClass({
 	},
 
 	handleInputBlur (event) {
-		if (document.activeElement.isEqualNode(this.refs.menu)) {
-			return;
-		}
+ 		let menuDOM = ReactDOM.findDOMNode(this.refs.menu);
```

Comment:
@gaastonsr findDOMNode is unnecessary in react 0.14 as this.refs.menu is a DOM Node. see https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html